analysis/study_definition.py

```diff
@@ -270,6 +270,14 @@
             "category": {"ratios": {"S": 0.6, "E": 0.1, "N + M": 0.3}, }
         },
     ),
+    has_msoa=patients.satisfying(
+        "NOT (msoa = '')",
+        msoa=patients.address_as_of(
+         "index_date",
+         returning="msoa",
+        ),
+        return_expectations={"incidence": 0.2}
```

Comment:
sorry I know this is my fault, but you should change this to 0.999 or 1 otherwise you'll filter out most of the dummy population

Reply:
oh perhaps not because filtering is ignored in the `population` step for dummy data

Reply:
I'll leave for now